DEVDOC.md

```diff
@@ -685,3 +685,34 @@ only the (transformed) capture commands needs to be replayed.
 If the user-requested replay does require to transform the initial commands,
 then the pre-warm replay is abandonned and a new replay of the transformed
 inital commands and then the transformed capture commands are executed.
+
+## Life of a perfetto trace (system profile trace)
+
+AGI heavily relies on the [perfetto](https://perfetto.dev/) system profiling
+framework to obtain, store and process profiling data. Hence we often refer to
+system profile traces as "perfetto traces". If you have to deal with perfetto
+code, make sure to refer to the [perfetto
+documentation](https://perfetto.dev/docs/).
+
+Perfetto is build into Android (since Android 9 Pie). In general, to take a
+perfetto trace, you can use the `perfetto` command-line tool on the device. See
+perfetto's web interface at https://ui.perfetto.dev/ and click on "Recording
+command" to see an example of how the `perfetto` command-line tool can be used
+to obtain a profiling trace.
+
+AGI does not use the `perfetto` command line tool. It directly interacts with
+perfetto's `traced` deamon running on the device. This deamon listens to the
+`/dev/socket/traced_consumer`, and AGI connects directly to this socket. The
+related AGI code is under `gapis/perfetto/client/`.
```

Comment:
There is another code path that uses the CLI to somehow activate the trace by catting things into files as far as I know.

Reply:
Thanks for this, I found it! I clarified the text accordingly.